openshift/ci-operator/knative-test-images/wathola-kafka-sender/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Do not edit! This file was generate via Makefile
+FROM openshift/origin-base
+
+ADD wathola-kafka-sender /usr/bin/wathola-kafka-sender
+ENTRYPOINT ["/usr/bin/wathola-kafka-sender"]
```

Comment:
In order to hav'em "in place", we need the confog for the `openshift/release` repo

Reply:
Right, but that would need to be done after this PR is merged, right?

Reply:
does not really matter.

/lgtm
/approve